source/manual/aws-iam-key-rotation.html.md

```diff
@@ -5,93 +5,141 @@ layout: manual_layout
 type: learn
 section: AWS
 owner_slack: "#govuk-developers"
-last_reviewed_on: 2019-10-31
+last_reviewed_on: 2020-01-20
 review_in: 6 months
 ---
 
 ## Overview
 
-GOV.UK uses [AWS IAM](https://docs.aws.amazon.com/en_pv/IAM/latest/UserGuide/id_users.html)
-accounts to manage applications access to our AWS infrastructure. The
-[Access Keys](https://docs.aws.amazon.com/en_pv/IAM/latest/UserGuide/id_credentials_access-keys.html)
-associated with these accounts need to be [rotated](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_RotateAccessKey)
-every 3 months.
+GOV.UK uses [AWS IAM] accounts to manage applications access to our AWS
+infrastructure. The [Access Keys] associated with these accounts need to be
+[rotated] every 3 months.
 
-## How to rotate Access Keys
+Rotating AWS IAM keys requires permissions which are available with the `admin`,
+`internal-admin`, `platformhealth-poweruser` or `poweruser` roles. If you can't
+assume one of these roles, you won't be able to rotate keys.
```

Comment:
What about changing this role so it doesn't use so many negatives.

> Only users who can assume these roles will be able to rotate keys.

Reply:
Yeah, can do